test/torchaudio_unittest/librosa_compatibility_test.py

```diff
@@ -165,13 +165,7 @@ def _load_audio_asset(*asset_paths, **kwargs):
 @unittest.skipIf(not LIBROSA_AVAILABLE, "Librosa not available")
 class TestTransforms(common_utils.TorchaudioTestCase):
     """Test suite for functions in `transforms` module."""
-    def assert_compatibilities(self, n_fft, hop_length, power, n_mels, n_mfcc, sample_rate):
-        common_utils.set_audio_backend('default')
-        path = common_utils.get_asset_path('sinewave.wav')
-        sound, sample_rate = common_utils.load_wav(path)
```

Comment:
The sample_rate variable on assert_compatabilities was not being used as it was overwritten here. I've removed it, and I removed it from the parameter dictionaries below.

Reply:
Sounds good.